pandas/io/excel/_odfreader.py

```diff
@@ -18,11 +18,13 @@ class _ODFReader(_BaseExcelReader):
     ----------
     filepath_or_buffer: string, path to be parsed or
         an open readable stream.
+    storage_options: dict (optional)
```

Comment:
need a versionadded tag

Reply:
Put into `pandas.io.excel._base._read_excel_doc`, so shows up for `pd.read_excel` (i.e., users are not expected to access these inner modules/classes)